### DISCUSSION TITLE
How to change color

### DISCUSSION THREAD

**Discussion by changshunxu520** [AtomMaterialUI/material-theme-issues#505] [Q&A]
When I use Project Title Bar Application Frame
Colorize Title Bar(New UI only)\Colorize Left Tool Window(New UI only) ....
Item shows a color，But I don’t like this color, how can I change it?
![325883257-90da158d-194b-4938-aa8f-9bccaa62679c](https://github.com/mallowigi/material-theme-issues/assets/16171624/7d7f44f0-2301-4fdf-93a0-07f18a0f9e1a)
![325884383-498ac7bf-8953-4789-a831-d66315eaa152](https://github.com/mallowigi/material-theme-issues/assets/16171624/932a2d69-ff02-44c5-8f52-cd8b94dc7f74)

**Comment by mallowigi** [comment]
Hi, you can change it from the "Per-project Title Bar" settings
